dds_web/api/user.py

```diff
@@ -383,14 +383,14 @@ def add_to_project(whom, project, role, send_email=True):
             AddUser.compose_and_send_email_to_user(whom, "project_release", project=project)
 
         flask.current_app.logger.debug(
-            f"{str(whom)} was associated with {str(project)} as Owner={is_owner}."
+            f"{str(whom)} was associated to the {str(project)} as a Researcher={is_owner}."
```

Comment:
Ah, actually what I meant is something like this: (removing the {is_owner} part)

```suggestion
            f"{str(whom)} was associated to the {str(project)} as a {'Project Owner' if is_owner else 'Researcher'}."
```